include/deal.II/grid/grid_generator.h

```diff
@@ -1907,6 +1907,34 @@ namespace GridGenerator
   flatten_triangulation(const Triangulation<dim, spacedim1> &in_tria,
                         Triangulation<dim, spacedim2> &      out_tria);
 
+  /**
+   * Convert a triangulation consisting only of hypercube cells
+   * (quadrilaterals, hexahedrons) to a triangulation only consisting of
+   * simplices (triangles, tetrahedra).
+   *
+   * @image convert_hypercube_to_simplex_mesh_visualization.png
```

Comment:
You'll have to write this as follows. I'm also adding a suggestion for some more words around it:
```suggestion
   * As an example, the following image shows how a set of three hexahedra
   * meshing one eighths of a sphere are subdivided into tetrahedra, and how
   * the curved surface is taken into account. Colors indicate how boundary indicators
   * are inherited:
   * @image html "convert_hypercube_to_simplex_mesh_visualization.png"
```